apps/test-suite/tests/FileSystemNext.ts

```diff
@@ -1,6 +1,7 @@
 'use strict';
 import * as FS from 'expo-file-system';
 import { File, Directory } from 'expo-file-system/next';
+import { join } from 'expo-file-system/src/next/path';
```

Comment:
Should we do this instead?
```suggestion
import { FileSystem } from 'expo-file-system/src/next';
FileSystem.join(...)
```

the current approach probably will play nicer with any future treeshaking we do, but the `path` naming is customary but also confusing (there's also `new File().path`)